services/pkg/buildbot/run.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+until /venv/bin/buildbot upgrade-master /buildbot; do
+    echo "upgrading buildmaster..."
```

Comment:
How safe is this to do as a purely unattended task?

Reply:
reading the [docs](https://docs.buildbot.net/current/manual/cmdline.html#upgrade-master), looks like it shouldn't be necessary to do on start unless we update, so it can/should be done manually

Reply:
I've made the upgrade a script `upgrade.sh` in the container that can be run in the container separately when doing upgrades